lib/bundle/dumper.rb

```diff
@@ -20,6 +20,7 @@ def self.dump_brewfile
       content << cask_before_formula
       content << BrewDumper.dump
       content << cask_after_formula
+      content << MacAppStoreDumper.dump
```

Comment:
I don't think we should make such assumption, which feels like you have to either use mas without bundle installed and use bundle without mas installed in order to separate manage homebrew and apps.